core/src/main/java/com/segment/analytics/kotlin/core/platform/Plugin.kt

```diff
@@ -136,4 +137,15 @@ abstract class DestinationPlugin : EventPlugin {
     final override fun execute(event: BaseEvent): BaseEvent? {
         return null
     }
+
+    internal fun isDestinationEnabled(event: BaseEvent?): Boolean {
+        // if event payload has integration marked false then its disabled by customer
+        val customerDisabled = when (event?.integrations?.getBoolean(key)) {
```

Comment:
how about the following?
```suggestion
        val customerEnable = event?.integrations?.getBoolean(key) ?: true
```
avoiding double negation for better readability, so when return 
```
    return (enabled && customerEnable)
```